theano/tensor/tests/test_opt.py

```diff
@@ -5350,6 +5418,8 @@ def test_reduction_opt(inputs, inputs_val, reduction_op,
                            (s1_val * s2_val * v_val * m_val).prod(), 2)
 
     def test_local_sum_prod_all_to_none(self):
+        # Julian: It appears that the opt local_sum_prod_mul_by_scalar
+        #         is never used in any of these tests...
```

Comment:
I thought it was tested on the function above, `test_local_sum_prod_mul_by_scalar`.
In any case, the test below seems good, maybe just mention "local_sum_prod_mul_by_scalar" in the comment.